src/main/java/ch/njol/skript/ScriptLoader.java

```diff
@@ -898,6 +898,22 @@ public static CompletableFuture<ScriptInfo> reloadScripts(Set<Script> scripts, O
 
 		return loadScripts(configs, openCloseable);
 	}
+
+	/**
+	 * Reloads all provided Scripts that could have potentially changed since their initial load.
+	 * This is tested by
+	 * @param scripts
+	 * @param openCloseable
+	 * @return
+	 */
+	public static CompletableFuture<ScriptInfo> reloadChangedScripts(Set<Script> scripts, OpenCloseable openCloseable) {
```

Comment:
I wonder if we would just want overload reloadScripts with a new parameter like `boolean onlyChanged`